.circleci/config.yml

```diff
@@ -57,6 +57,33 @@ jobs:
       - run:
           name: Publish @next
           command: npm publish --tag next --access public
+  docker_next:
+    <<: *defaults
+    steps:
+      - attach_workspace:
+          at: ~/cli-microsoft365
+      - setup_remote_docker:
+          version: 19.03.13
+      - run:
+          name: Extract version from package
+          command: |
+            echo "export PACKAGE_VERSION=$(node -p "require('./package.json').version")" >> $BASH_ENV
```

Comment:
This line leads to incorrect results, because the changes to the `package.json` file, from the `publish_next` job are not persisted to the workspace. As a result, this line returns the original package version from `package.json` without the commit hash. For this to work, we'd need to extend the `publish_next` step with persisting changes to the workspace.